<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourcesGui</class>
 <widget class="QWidget" name="ResourcesGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget_resources">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>341</width>
         <height>351</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_resources">
        <item>
         <widget class="QPushButton" name="pushButton_changeColor">
          <property name="text">
           <string>Change Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_createMessage">
          <property name="text">
           <string>Create Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_viewMessage">
          <property name="text">
           <string>View Message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteMessage">
          <property name="text">
           <string>Delete Message</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLineEdit" name="lineEdit_text">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>240</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_saveMessage">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>290</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Message</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_title">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>170</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_text">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>240</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Text:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>331</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_view">
        <item>
         <widget class="QLabel" name="label_titleView">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_titleView"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_view">
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>280</y>
         <width>100</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_displayText">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>240</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_titlesOutput">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>331</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>143</x>
         <y>240</y>
         <width>331</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_deleteMessage">
        <item>
         <widget class="QLabel" name="label_titleDeleteMessage">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputTitle_3"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_allTitles">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>311</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
